src/pybind/mgr/dashboard/tests/test_docs.py

```diff
@@ -112,3 +114,117 @@ def test_gen_paths_all(self):
     def test_gen_tags(self):
         outcome = Docs()._gen_tags(False)[0]
         self.assertEqual({'description': 'Group description', 'name': 'FooGroup'}, outcome)
+
+
+class TestEndpointDocWrapper(unittest.TestCase):
+    def test_wrong_param_types(self):
+        with self.assertRaises(Exception):
+            EndpointDoc(description=False)
+        with self.assertRaises(Exception):
+            EndpointDoc(group=False)
```

Comment:
New tests 🥳 ! Just for your own satisfaction and general praise, would you mind printing the before & after coverage stats for these files?

Reply:
![image](https://user-images.githubusercontent.com/30913090/116228167-36e57c00-a755-11eb-82d7-f51c564ae57f.png)
Left is before and right After the changes
exceptions.py decreased because of the new decorator I think